### PR TITLE
[javascript] Add CVE-2017-16138

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by CermakM** [comment]
@msrb Reviewed

File: database/javascript/2017/16138.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    The mime module is vulnerable to regular expression denial of service when a mime lookup is performed on untrusted user input.
+cvss_v2: 5.0
+references:
+    - https://github.com/broofa/node-mime/issues/167
+    - https://nodesecurity.io/advisories/535
+
+affected:
+    - name: mime
+      version:
+        - "<1.4.1"
```
Comment (msrb) [inline database/javascript/2017/16138.yaml]: This syntax is not supported by VictimsDB: https://github.com/victims/victims-cve-db#version-string-common

Comment (CermakM) [reply database/javascript/2017/16138.yaml]: Ah right.. forgot about this thing. Thanks for being thorough!

**Comment by CermakM** [comment]
@msrb Fixed

**Pull request merged**